docs/index.md

```diff
@@ -440,6 +440,32 @@ No Context
 
 No Context 
 
+### sync-error
```

Comment:
Should this (or general error scenarios) be described more generally? IE: is the the only operationOutcome that can be sent or is OperationOutcome in general the way this would be done?

Reply:
Hey Jenni, I just created #77 to track your comment to continue moving ahead with this PR. If you feel this needs to be addressed before merging, will you block or let me know?